### PR TITLE
DESKTOP VIEW 

### PR DESCRIPTION
**Pull request by aneallaryea100**
# Desktop view added to the project to make it responsive.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,45 +21,70 @@
...
+                 <a href="#" class="my-logo">Aneal Laryea</a>
+                <!--<a href="#"><img src="assets/image/menn1.png" alt="menu"></a> -->
+                <!-- <div class="logo">
+                    <a href="#" class="my-logo">Aneal Laryea</a>
+                  </div> -->
+                
+                <ul class="sects">
+                    <li class="port"><a href="#">Portfolio</a></li>
+                    <li class="aabout mp dbx"><a href="#">About</a></li>
+                    <li class="contact mp dbx"><a href="#">Contact</a></li>
```
Comment (SajjadAhmad14) [inline index.html:40]: There is a design issue with the navigation on the desktop view. Moreover kindly use the same font color for navigation items as in the design which is `Inter`.
nav

File: index.html [hunk 2]
```diff
@@ -21,45 +21,70 @@
...
+            <div class="mmed">
+                <ul class="ussm">
                     <li class="ss"><img src="assets/image/github.png" alt="github icon"></li>
                     <li class="ss"><img src="assets/image/linkdin.png" alt="linkdin icon"></li>
-                    <li class="ss"><i class="fa-brands fa-angellist"></i></li>
-                    <li class="ss"><i class="fa-brands fa-twitter"></i></li>
-                    <li class="ss"><i class="fa-solid fa-m"></i></li>
+                    <li class="ss"><img src="assets/image/angel1.png" alt="angelist icon"></li>
+                    <li class="ss"><img src="assets/image/twit1.png" alt="twitter icon"></li>
+                    <li class="ss"><img src="assets/image/med1.png" alt="medium icon"></li>
```
Comment (SajjadAhmad14) [inline index.html]: These social icons should be linked to social profiles. Kindly use `a` tags to make them links.
head-links

File: index.html [hunk 3]
```diff
@@ -183,7 +215,7 @@ <h2 class="title-text">My Recent Works</h2>
            </ul>
            </div>
            <div class="actz">
-               <button class="clicks btnn" type="button">See Project</button>
+               <button class="clicks btnn dp" type="button">See Project</button>
            </div>
            
        </div>
```
Comment (SajjadAhmad14) [inline index.html:221]: Please create a footer for the desktop view as it is defined in the design.
footer-o

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Aneal, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (aneallaryea100) [reply index.html:40]: * Design issue fix. The same font color was used and the same font family was applied to navigation items.

File: index.html (hunk 2 above)
Comment (aneallaryea100) [reply index.html]: * a tags linked to social profiles.

File: index.html (hunk 3 above)
Comment (aneallaryea100) [reply index.html:221]: Footer created for the desktop view.

**Comment by aneallaryea100** [comment]
# Desktop View
* Footer added to the project
* Social icons have A tag
* Design color and font applied to navigation items

**Review by rustam-tolipov — APPROVED** [review]
## Hi @aneallaryea100   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**